chpl/chpl-resources/src/main/resources/errors.properties

```
@@ -669,7 +669,7 @@ changeRequest.multipleAcbs=User with access to multiple certification bodies can
 
 realWorldTesting.upload.chplProductNumberInvalid=CHPL Product Number is either missing or invalid.
 realWorldTesting.upload.realWorldTestingTypeInvalid=Real World Testing Type is either missing or invalid.
-realWorldTesting.upload.lastCheckedDateInvalid=Last Checked Date is either missing or invalid.
-realWorldTesting.upload.url=URL is either missing or invalid.
-realWorldTesting.upload.dataNotChanged=Uploaded data matches the listing's current Real World Testing information.
+realWorldTesting.upload.lastCheckedDateInvalid=Last Checked Date is invalid.
+realWorldTesting.upload.url=URL is invalid.
+realWorldTesting.upload.dataNotChanged=WARNING: Uploaded data matches the listing's current Real World Testing information.  Listing not updated.
```

Can we remove the double spaces?
```suggestion
realWorldTesting.upload.dataNotChanged=WARNING: Uploaded data matches the listing's current Real World Testing information. Listing not updated.
```